docs/reference/api/rtos/rtos.md

```diff
@@ -51,21 +55,6 @@ Each `Thread` can wait for signals and be notified of events:
 
 [![View code](https://www.mbed.com/embed/?url=https://os.mbed.com/teams/mbed_example/code/rtos_signals/)](https://os.mbed.com/teams/mbed_example/code/rtos_signals/file/476186ff82cf/main.cpp)
 
-##### Status and error codes
-
-The CMSIS-RTOS functions will return the following statuses:
-
-- `osOK`: function completed; no event occurred.
-- `osEventSignal`: function completed; signal event occurred.
-- `osEventMessage`: function completed; message event occurred.
-- `osEventMail`: function completed; mail event occurred.
-- `osEventTimeout`: function completed; timeout occurred.
-- `osErrorParameter`: a mandatory parameter was missing or specified an incorrect object.
-- `osErrorResource`: a specified resource was not available.
-- `osErrorTimeoutResource`:  a specified resource was not available within the timeout period.
-- `osErrorISR`: the function cannot be called from interrupt service routines (ISR).
-- `osErrorISRRecursive`: function called multiple times from ISR with same object.
-- `osErrorPriority`: system cannot determine priority or thread has illegal priority.
-- `osErrorNoMemory`: system is out of memory; it was impossible to allocate or reserve memory for the operation.
-- `osErrorValue`: value of a parameter is out of range.
-- `osErrorOS`: unspecified RTOS error - runtime error but no other error message fits.
+##### Status and Error codes
+
+The Arm Mbed RTOS layer handles RTOS errors and report them using the Mbed OS error handling system. Please see the list of error codes in the [error handling](/docs/development/reference/error-handling.html) documentation for more information on RTOS errors reported.
```

Comment:
dont describe layers, The RTOS errors codes are documented as part of OS errors...